stm/vernac_classifier.ml

```diff
@@ -176,7 +176,7 @@ let classify_vernac e =
     (* These commands alter the parser *)
     | VernacOpenCloseScope _ | VernacDelimiters _ | VernacBindScope _
     | VernacInfix _ | VernacNotation _ | VernacNotationAddFormat _
-    | VernacSyntaxExtension _
+    | VernacSyntaxExtension _ | VernacDeclareCustomEntry _
```

Comment:
I don't think declaring an entry changes the parser in any visible way. It should not be `VtNow`.